docker-bake.hcl

```diff
@@ -22,7 +22,19 @@ group "linux-ppc64le" {
   targets = []
 }
 
-# update this to use a newer build number of the jenkins/agent image
+variable "IMAGE_TAG" {
+  default = "3063.v26e24490f041-1"
+}
+
+variable "REMOTING_VERSION" {
+  default = split("-", "${IMAGE_TAG}")[0]
+}
+
+variable "BUILD_NUMBER" {
```

Comment:
there's already a build number variable in this file

Reply:
this one is the `inbound-agent` build number (It took me 8hours in the plane to fully get it :D ).

Let me add a comment on this one.

Reply:
you sure?
![image](https://user-images.githubusercontent.com/21194782/193012738-370371a6-ff1d-40a6-a45f-7897868d9590.png)

Reply:
Oooh good catch: this 2nd occurence is the result of me messing up the rebase.

Should be fixed now